php/class-media-library.php

```diff
@@ -41,7 +41,7 @@ public function setup() {
 		$this->handle = add_menu_page(
 			__( 'Cloudinary Media Library', 'cloudinary' ),
 			__( 'Cloudinary DAM', 'cloudinary' ),
-			'manage_options',
+			Utils::user_can( 'manage_dam' ),
```

Comment:
This will result in passing a boolean into the capability param for the `add_menu_page()` adding a `true` will result in the `map_meta_cap()` defaulting to `remove_users` Which will allow admins to view it.
But if `manage_dam` is filtered to `upload_files` so that editors can use it, it will not work.

Perhaps something like this would be better?
```suggestion
			Utils::user_can( 'manage_dam' ) ? 'exist' : false,
```